### PR TITLE
Pwa enhancement

### PR DESCRIPTION
**Pull request by bhardwajaditya**

### CODE REVIEW

File: imports/message-read-receipt/client/readReceipts.css [hunk 1]
```diff
@@ -5,8 +5,12 @@
 }
 
 .read-receipt .rc-icon {
-	height: 0.8em;
-	width: 0.8em;
+	height: 1em;
+	width: 1em;
+}
+/* Overwriting the color-component-color class defined in colors.less */
```
Comment (sampaiodiego) [inline imports/message-read-receipt/client/readReceipts.css]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

Comment (sampaiodiego) [inline imports/message-read-receipt/client/readReceipts.css]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

Comment (sampaiodiego) [inline imports/message-read-receipt/client/readReceipts.css]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

Comment (sampaiodiego) [inline imports/message-read-receipt/client/readReceipts.css]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

File: app/theme/client/imports/components/header.css [hunk 2]
```diff
@@ -246,6 +246,7 @@
 
 	&__toggle-favorite {
 
+		margin: 0 0.75rem;
```
Comment (sampaiodiego) [inline app/theme/client/imports/components/header.css]: Codacy Issue found: [Unexpected empty line before property "margin" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

File: imports/message-read-receipt/client/readReceipts.css (hunk 1 above)
Comment (sampaiodiego) [inline imports/message-read-receipt/client/readReceipts.css]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

File: app/theme/client/imports/components/header.css (hunk 2 above)
Comment (sampaiodiego) [inline app/theme/client/imports/components/header.css]: Codacy Issue found: [Unexpected empty line before property "margin" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

File: imports/message-read-receipt/client/readReceipts.css (hunk 1 above)
Comment (sampaiodiego) [inline imports/message-read-receipt/client/readReceipts.css]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

File: app/theme/client/imports/general/base_old.css [hunk 3]
```diff
@@ -360,7 +360,7 @@
 
 		& > div {
 
-			width: 60%;
+			width: 55%;
```
Comment (sampaiodiego) [inline app/theme/client/imports/general/base_old.css]: Codacy Issue found: [Unexpected empty line before property "width" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

File: app/theme/client/imports/components/header.css (hunk 2 above)
Comment (sampaiodiego) [inline app/theme/client/imports/components/header.css]: Codacy Issue found: [Unexpected empty line before property "margin" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

File: imports/message-read-receipt/client/readReceipts.css (hunk 1 above)
Comment (sampaiodiego) [inline imports/message-read-receipt/client/readReceipts.css]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

File: app/theme/client/imports/general/base_old.css (hunk 3 above)
Comment (sampaiodiego) [inline app/theme/client/imports/general/base_old.css]: Codacy Issue found: [Unexpected empty line before property "width" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

File: app/theme/client/imports/components/header.css (hunk 2 above)
Comment (sampaiodiego) [inline app/theme/client/imports/components/header.css]: Codacy Issue found: [Unexpected empty line before property "margin" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5127980)

**Comment by ggazzo** [comment]
Thank you for your contribution. However, we have had several changes and I believe that some of these improvements have already been implemented.

**Pull request closed without merging**